src/raphson.jl

```diff
@@ -96,7 +97,7 @@ function perform_step!(cache::NewtonRaphsonCache{true})
 
     # Line Search
     α = perform_linesearch!(cache.lscache, u, du)
-    @. u = u - α * du
+    axpy!(-α, du, u)
```

Comment:
does this have appropriate generic fallbacks?

Reply:
Done